src/legacy/core_plugins/kibana/public/dashboard/panel/dashboard_panel.tsx

```diff
@@ -17,127 +17,160 @@
  * under the License.
  */
 
-import React from 'react';
-import PropTypes from 'prop-types';
-import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
+
+import { EuiPanel } from '@elastic/eui';
 import classNames from 'classnames';
 import _ from 'lodash';
-
-import { PanelHeader } from './panel_header';
+import React from 'react';
+import { ContainerState, Embeddable, EmbeddableFactory } from 'ui/embeddable';
+import { EmbeddableActions, EmbeddableErrorAction } from '../actions';
+import { PanelState } from '../selectors';
 import { PanelError } from './panel_error';
+import { PanelHeader } from './panel_header/panel_header';
+
+interface DashboardPanelProps {
+  intl: any;
+  viewOnlyMode: boolean;
+  onPanelFocused: (panel: string) => {};
+  onPanelBlurred: (panel: string) => {};
+  error: string | object;
+  destroy: () => void;
+  containerState: ContainerState;
+  embeddableFactory: EmbeddableFactory;
+  embeddableStateChanged: (
+    embeddableStateChanges: import('ui/embeddable/types').EmbeddableState
+  ) => {};
+  embeddableIsInitializing: EmbeddableActions;
+  embeddableIsInitialized: (
+    embeddableIsInitialized: import('ui/embeddable/embeddable').EmbeddableMetadata
```

Comment:
I think the intake ci is choking on these inline imports.  I know we fixed them, but then ended up throwing the changes away and didn't go back around to it. 

```
16:51:13 [21:51:13] →  I18N ERROR  Error in src/legacy/core_plugins/kibana/public/dashboard/panel/dashboard_panel.tsx
16:51:13 Error: Unexpected token (42:28):
16:51:13   embeddableFactory: EmbeddableFactory;
16:51:13   embeddableStateChanged: (
16:51:13     embeddableStateChanges: import('ui/embeddable/types').EmbeddableState
16:51:13   ) => {};
16:51:13   embeddableIsInitializing: EmbeddableActions;
16:51:13 ERROR  I18N ERROR  Error in src/legacy/core_plugins/kibana/public/dashboard/panel/dashboard_panel.tsx
16:51:13       Error: Unexpected token (42:28):
16:51:13         embeddableFactory: EmbeddableFactory;
16:51:13         embeddableStateChanged: (
16:51:13           embeddableStateChanges: import('ui/embeddable/types').EmbeddableState
16:51:13         ) => {};
16:51:13         embeddableIsInitializing: EmbeddableActions;
16:51:13 Warning: non-zero exit code 1 Use --force to continue.
16:51:13 
```

Look at the intake failures first, it's possible something being caught in there is causing the later selenium failures as well, but the error output on the intake one is easier to pinpoint what exactly the issue is.

Reply:
Awesome! I think I was able to rebase the commits from my feature branch over that should have the import changes in dashboard panel.